docs/tutorials/first-plugin.md

```diff
@@ -0,0 +1,208 @@
+# How to create your first UCRM plugin
+
+This is a step by step guide for creating your first simple UCRM plugin.
+
+## 1) Get your environment ready
+First step is to prepare your development environment. You should have to following software installed:
+- PHP 7.1 - https://secure.php.net/downloads.php
```

Comment:
7.1 or 7.2

Reply:
Nope, there is only UCRM with PHP 7.1 released. When we release version with 7.2, I'll update the tutorial. Since this is beginner guide, I don't want to say to install 7.2, because if they use features from it, they will not work in current UCRM.

Reply:
7.3 has some minor bugs in the RC still, we don't want to encourage that.